test/integration/integration_test.go

```diff
@@ -21,43 +22,15 @@ import (
 	"k8s.io/apimachinery/pkg/util/wait"
 )
 
-var operatorNamespace string
-var clusterName string
+func TestMultipleIngresses(t *testing.T) {
```

Comment:
I was using subtests deliberately to support setup/teardown around (theoretical) multiple test cases involving the same operator deployment. Now every test case in this file will spin up a new competing operator goroutine.